src/System.Net.ServicePoint/ref/project.json

```diff
@@ -0,0 +1,19 @@
+{
+  "dependencies": {
+    "System.Collections.Specialized": "4.0.0",
+    "System.IO": "4.1.0",
```

Comment:
going forward, it was decided to use the latest packages in the refs as well.. 
cc @joperezr